tests/lib/snaps/test-snapd-autopilot-consumer/wrapper

```diff
@@ -0,0 +1,3 @@
+export GI_TYPELIB_PATH=$SNAP/usr/lib/girepository-1.0:$(ls -d $SNAP/usr/lib/*/girepository-1.0)
+
+$SNAP/usr/bin/python3 $SNAP/provider.py
```

Comment:
I see, okay. Would be good to solve these cases over time, but not a problem for this PR.